src/HtmlElement.php

```diff
@@ -122,7 +122,7 @@ public function __toString()
             $result .= ' ' . $key . '="' . $value . '"';
         }
 
-        if ($this->contents) {
+        if (!is_null($this->contents) and $this->contents != '') {
```

Comment:
Ugh I'm probably using `null` somewhere I shouldn't be.  /facepalm

I'll look into this.